app/services/search/feed_content.rb

```diff
@@ -0,0 +1,25 @@
+module Search
+  class FeedContent < Base
+    INDEX_NAME = "feed_content_#{Rails.env}".freeze
+    INDEX_ALIAS = "feed_content_#{Rails.env}_alias".freeze
+    MAPPINGS = JSON.parse(File.read("config/elasticsearch/mappings/feed_content.json"), symbolize_names: true).freeze
+
+    class << self
+      private
+
+      def index_settings
+        if Rails.env.production?
+          {
+            number_of_shards: 10,
```

Comment:
There's not a solid "rule of thumb" for something like this, right? It's arbitrary?

Reply:
Yeah, it's based on how many I think will last us for a while but also not use a ton of resources right off the bat